app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -264,6 +286,14 @@ export default {
       const stateInbox = this.inbox;
       return stateInbox.name || '';
     },
+    assignableAgents() {
+      return this.$store.getters['inboxAssignableAgents/getAssignableAgents'](
+        this.inbox.id
+      );
+    },
+  },
+  mounted() {
+    this.$store.dispatch('inboxAssignableAgents/fetch', [this.inbox.id]);
```

Comment:
> This is not the correct place for this call. It would create unnecessary calls. The fetch call should be fired only if the context menu is opened.
> 
> <img alt="Screenshot 2022-07-20 at 3 00 29 PM" width="1125" src="https://user-images.githubusercontent.com/2246121/179949151-fb966306-b57e-4371-a139-ba5db124fac5.png">

I thought about this, but the way the submenu is show is purely via css, hence I needed a way to hook the api call on a mounted method.

Will figure out a way to handle this some other way.